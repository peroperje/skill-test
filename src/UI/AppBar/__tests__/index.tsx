import React from 'react';
import { create } from 'react-test-renderer';

import { AppBar } from '../index';

describe('AppBar', () => {
  it('Snapshot', () => {
    const component = create(<AppBar />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
