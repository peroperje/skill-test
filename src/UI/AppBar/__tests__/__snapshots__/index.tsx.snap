// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar Snapshot 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed"
/>
`;
